stdlib/@python2/abc.pyi

```diff
@@ -18,7 +18,7 @@ class ABCMeta(type):
     def __init__(self, name: str, bases: Tuple[type, ...], namespace: Dict[Any, Any]) -> None: ...
```

Comment:
```suggestion
    def __init__(self, name: str, bases: Tuple[type, ...], namespace: Dict[str, Any]) -> None: ...
```

Reply:
@Akuli do these need to be strings also?

Reply:
Honestly I don't care much about Python 2 stubs, but feel free to change it if you do.